pom.xml

```diff
@@ -83,11 +83,6 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <configuration>
-          <excludes>
-            <exclude>org/jitsi/impl/neomedia/transform/srtp/CryptoBenchmark.java</exclude>
-          </excludes>
-        </configuration>
       </plugin>
```

Comment:
I'm working on merging this PR so please don't touch it. But I want to state that I don't like it because it's incomplete and I have to spend non-trivial merge time on a non-issue (or at least an issue with very low priority). (1) Since you've removed the configuration tag, the whole plugin tag has become obsolete. (2) The discussion of whether the Ant build script is necessary is a different question out of the scope of this PR so, since you've removed the exclusion in the Maven file, it should be removed from the Ant file as well. Again, I'll do these so that I can merge this PR and be done with it.